package.json

```diff
@@ -55,7 +55,7 @@
   "homepage": "https://github.com/pagopa/io-backend#readme",
   "dependencies": {
     "@azure/storage-queue": "^12.0.0",
-    "@pagopa/io-spid-commons": "^4.4.0",
+    "@pagopa/io-spid-commons": "^4.7.0",
```

Comment:
it's ok with this pr, but since there are some changes in io-spid-commons which are unrelated to event tracking, that would be better to make a separate request with only the upgrade in order to have a changelog that clearly list the changes (which is the one you'll see before deploy)